tools/maxquant/maxquant.xml

```diff
@@ -309,51 +285,22 @@ sample:$j
     </macro>
   </macros>
   <inputs>
-    <conditional name="analysis_type">
-      <param name="type" type="select" label="Analysis Type" help="The wrapper has not yet implemented multiple groups with different parameters">
-        <option value="sg_fractions" selected="true">Single Group - Fractions of One Sample</option>
-        <option value="sg_samples">Single Group - Single File Samples</option>
-        <option value="sg_both">Single Group - Multiple Fractions and Samples</option>
-        <option value="mgs_fractions">Multi-Group (Identical Parameters) - Fractions of One Sample</option>
-        <option value="mgs_samples">Multi-Group (Identical Parameters) - Single File Samples</option>
-        <option value="mgs_both">Multi-Group (Identical Parameters) - Multiple Fractions and Samples</option>
-      </param>
-      <when value="mgs_fractions">
-        <repeat name="groups" title="Group">
-          <expand macro="input_param" />
-        </repeat>
-	      <expand macro="group_params" />
-      </when>
-      <when value="mgs_samples">
-        <repeat name="groups" title="Group">
-          <expand macro="input_param" />
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-      <when value="mgs_both">
-        <repeat name="groups" title="Group">
-          <repeat name="samples" title="Sample">
-            <expand macro="input_param" />
-          </repeat>
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_fractions">
-        <expand macro="input_param" />
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_samples">
-        <expand macro="input_param" />
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_both">
-        <repeat name="samples" title="Sample">
-          <expand macro="input_param" />
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-    </conditional>
+    <repeat name="sample" title="Sample">
```

Comment:
Mateus;
Why removing these options if they are working, even if the use-case is very rare?